frappe/integrations/doctype/integration_request/integration_request.py

```diff
@@ -20,3 +22,11 @@ def update_status(self, params, status):
 		self.status = status
 		self.save(ignore_permissions=True)
 		frappe.db.commit()
+
+	def process_response(self, ref_field, response):
```

Comment:
`process_response` is generic and does not convey what it is actually doing.

This is basically handling "success" and "failure" actions. This should be done with two methods: `handle_success` and `handle_failure`.

Reply:
added please check again